### PR TITLE
Issue type

### PR DESCRIPTION
**Pull request by zvxr**

### CODE REVIEW

File: templates/charts_template.json [hunk 1]
```diff
@@ -30,7 +30,75 @@
...
+                },
+                "start_on": {
+                    "type": "date",
+                    "format": "yyyy-MM-dd HH:mm:ss",
+                    "store": "no",
+                    "index": "analyzed"
+                },
+                "end_on": {
+                    "type": "date",
+                    "format": "yyyy-MM-dd HH:mm:ss",
```
Comment (CarlosMecha) [inline templates/charts_template.json]: Can we use the standard `yyyy-MM-ddTHH:mm:ss`? Is the default format for ElasticSearch and we are already using it for the catalog.

File: templates/charts_template.json [hunk 2]
```diff
@@ -30,7 +30,75 @@
...
+                "release_id": {
+                    "type": "long",
+                    "store": "yes",
+                    "index": "not_analyzed"
+                },
+                "release_title": {
+                    "type": "string",
+                    "store": "no",
                     "index": "analyzed"
                 }
```
Comment (CarlosMecha) [inline templates/charts_template.json]: We also need the timestamp order :+1:

Comment (zvxr) [inline templates/charts_template.json]: added

File: templates/charts_template.json (hunk 1 above)
Comment (zvxr) [inline templates/charts_template.json]: updated -- we should def adopt this going forward -- I think there's a couple places in CB might be going up as `yyyy-MM-dd HH:mm:ss`

**Pull request merged**